code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -49,7 +49,7 @@
 					new /obj/item/weapon/gun/projectile/pistol(get_turf(H))
 					new /obj/item/gun_part/silencer(get_turf(H))
 				if("cannon")
-					new /obj/item/weapon/gun/energy/lasercannon(get_turf(H))
+					new /obj/item/weapon/gun/energy/laser/cannon(get_turf(H))
```

Comment:
The poll was closed prematurely. I've been online all day and exactly one person was in favor of it. Things like this should have at least a 2/3 majority, not slip by because it squeaked past by four votes. Especially since many people didn't wholly understand what they were voting for.